mne/utils.py

```diff
@@ -510,6 +510,21 @@ def _reject_data_segments(data, reject, flat, decim, info, tstep):
     return data, drop_inds
 
 
+def _get_inst_data(inst):
```

Comment:
Yeah @choldgraf instance methods aren't supposed to have copy params anymore, it's all .copy().